### PR TITLE
The Fetch API adds the "fetch" global.

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

**Comment by caitp** [comment]
this is true, lgtm

File: tests/unit/fixtures/browser.js [hunk 1]
```diff
@@ -25,3 +25,12 @@ var comment = new Comment("");
 var df = new DocumentFragment();
 var range = new Range();
 var text = new Text("");
+
+// Fetch example
```
Comment (julienw) [inline tests/unit/fixtures/browser.js]: maybe I should add that this comes from MDN https://developer.mozilla.org/en-US/docs/Web/API/GlobalFetch ?

**Comment by silvenon** [comment]
:+1:

**Comment by julienw** [comment]
I added the link to MDN.

File: tests/unit/fixtures/browser.js (hunk 1 above)
Comment (julienw) [inline tests/unit/fixtures/browser.js]: done :)

**Comment by rwaldron** [comment]
Thanks @julienw!

**Pull request closed without merging**